geode-benchmarks/build.gradle

```diff
@@ -93,6 +93,7 @@ task benchmark(type: Test) {
   }
   systemProperty 'withSsl', project.hasProperty('withSsl')
   systemProperty 'withSniProxy', project.hasProperty('withSniProxy')
+  systemProperty 'withClusterTopology', project.hasProperty('withSniProxy')
```

Comment:
I think you want `project.hasProperty('withSniProxy')`.

Reply:
I think @pivotal-jbarrett meant to say: `project.hasProperty('withClusterTopology')`

Reply:
yes, I'll fix that